styles.css

```
@@ -115,4 +115,27 @@
         border-radius: 20px;
         border: 0;
     }
+
+    footer{
+        height: 132px;
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        flex-wrap: wrap;
+    }
+
+    .social{
+        display: flex;
+        justify-content: space-between  ;
```

se te fue un espacio

```suggestion
        justify-content: space-between;
```